### PR TITLE
add supported tests matrix

### PR DESCRIPTION
**Pull request by maggiepint**
More clearly documenting what OS and Java versions we are currently actively testing against.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -137,6 +137,15 @@ If you would like to become an active contributor to this project please follow
...
 5. Create new Pull Request
 
+## Tested Operating Systems and Java Versions
+
+Currently the tests in this repository are running against:
+
+|                      | Linux (Ubuntu 16.04) | MacOS 10.13 | Windows Server 2016 |
+|----------------------|----------------------|-------------|---------------------|
+| **Java 8**           |  x                   |      x      |          x          |
+| **Java LTS**         |  x                   |      x      |          x          |
```
Comment (JonathanGiles) [inline README.md]: I prefer to say "Java 11 (LTS)" rather than "Java LTS".

**Comment by maggiepint** [comment]
@Azure/azure-sdk-eng specified client libraries for now. This look good?

**Comment by joshfree** [comment]
@maggiepint I know this is an old PR, but do you still want to resolve the conflict (move the table to near the bottom of the readme) and merge?

**Comment by weshaggard** [comment]
Closing this PR as it is stale. We can reopen it again if we feel it is interesting still.

**Pull request closed without merging**